tapas-dl.py

```diff
@@ -51,9 +52,14 @@ def check_path(path, slash=True, fat=False):
 parser.add_argument('-f', '--force', action="store_true", help='Disables updater.')
 parser.add_argument('-v', '--verbose', action="store_true", help='Enables verbose mode.')
 parser.add_argument('-c', '--restrict-characters', action="store_true", help='Removes \'? < > \\ : * | " ^\' from file names')
+parser.add_argument('-o', type=str, nargs='?', default="", dest='baseDir', metavar='C:\\',
```

Comment:
Would be nice to also use '--output-dir', but that's only a small consistency nitpick.

Reply:
That makes good sense! 
I am not sure how to incorporate that change. I have committed/pushed it to my branch.